haystack/nodes/preprocessor/preprocessor.py

```diff
@@ -444,3 +459,52 @@ def _find_longest_common_ngram(
             # no common sequence found
             longest = ""
         return longest if longest.strip() else None
+
+    def _split_sentences(self, text: str) -> List[str]:
+        """
+        Tokenize text into sentences.
+        :param text: str, text to tokenize
+        :param language: str, language of text
+        :param tokenizer_model_path: Path, path to tokenizer model
+        :return: list[str], list of sentences
+        """
+        sentences = []
+
+        language_name = iso639_to_nltk.get(self.language)
+
+        # Try to load a custom model from 'tokenizer_model_path'
+        if self.tokenizer_model_folder:
```

Comment:
Let's make the check for None explicit.

```suggestion
        if self.tokenizer_model_folder is not None:
```